apps/api/lib/utils/isValidBase64.ts

```diff
@@ -0,0 +1,17 @@
+export function isValidBase64(input: string): boolean {
+  try {
```

Comment:
```suggestion
  const base64Regex = /^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$/;

    try {
```

Can potentially use a regex here for strict checking for base64
This pattern assumes a character set of A-Z, a-z, 0-9, +, / and possibly ending with = or ==.